tests/helpme/test_integrate.py

```
@@ -178,15 +178,58 @@ def test_function_integration(np_rng):
         bezier.integrate.parametric_function(volume_function),
     )
+
+    # try bsplines
+    bspline = bezier.bspline
+    assert np.allclose(
+        [bspline.integrate.volume(), col1_factor * bspline.integrate.volume()],
+        bspline.integrate.parametric_function(volume_function),
+    )
+
+
+def test_physical_function_integration(np_rng):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)